docs/csharp/tutorials/intro-to-csharp/list-collection.yml

```diff
@@ -162,7 +162,7 @@ items:
     > This online coding experience is in preview mode. If you encounter problems, please report them [on the dotnet/try repo](https://github.com/dotnet/try/issues).
     
 - content: |
-    You've completed the list tutorial. This tutorial is the final interactive tutorial. You can continue these tutorials on your own development environment. Learn the basics of [local development](local-environment.md) and then pick a tutorial. You can try this same exercise, move directly to the next tutorial, or start again at with the [numbers in C#](numbers-in-csharp-local.md) tutorial.
+    You've completed the list tutorial. You've completed the introduction to C# interactive tutorials. You can click the **Next** link below to start the **string interpolation interactive tutorial**, which is more advanced. Or, you can visit the [.NET site](https://www.microsoft.com/net/learn/dotnet/hello-world-tutorial) to download the .NET Core SDK, create a project on your machine, and keep coding. The "Keep Learning" step brings you back to these tutorials.
```

Comment:
- completed the --> now completed all of the 
- There is no Next link, nor a link to the string interpolation tutorial.
- no comma after "Or"
Again, "Keep learning" doesn't appear